backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/steps/Sanitize.kt

```diff
@@ -0,0 +1,100 @@
+package app.ehrenamtskarte.backend.stores.importer.steps
+
+import app.ehrenamtskarte.backend.common.STATE
+import app.ehrenamtskarte.backend.stores.geocoding.isCloseTo
+import app.ehrenamtskarte.backend.stores.geocoding.queryFeatures
+import app.ehrenamtskarte.backend.stores.importer.PipelineStep
+import app.ehrenamtskarte.backend.stores.importer.types.AcceptingStore
+import kotlinx.coroutines.runBlocking
+import org.geojson.Feature
+import org.geojson.Point
+import org.slf4j.Logger
+
+const val DISTANCE_THRESHOLD_IN_KM = 1.0
+
+// Postal code lookup fails/does not really make sense for a "Postfach"
+const val STREET_EXCLUDE_PATTERN = "Postfach"
+
+class Sanitize(private val logger: Logger) : PipelineStep<List<AcceptingStore>, List<AcceptingStore>>() {
+
+    override fun execute(input: List<AcceptingStore>): List<AcceptingStore> = runBlocking {
+        val stateBbox = queryFeatures(listOf(Pair("state", STATE))).first().bbox
+
+        input.map { it.sanitize() }.filter {
+            if (!it.isInBoundingBox(stateBbox)) {
```

Comment:
Is this executed after the long/lat has been corrected using geocoding? I think it makes sense to do a bounding box check only after correcting the long/lat.

Reply:
yes, its done afterwards. Correcting the coordinates is done via `it.sanitize()`